packages/react-examples/README.md

```diff
@@ -1,47 +1,77 @@
 # @fluentui/react-examples
 
-**Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+This package contains examples and documentation site content for `@fluentui/react` v8 and related packages, including `@fluentui/react-charting` and `@fluentui/react-experiments`.
 
-This package is only meant to be consumed by variants of the Fluent UI documentation site and therefore has **NO GUARANTEE OF API STABILITY**.
```

Comment:
This note was from back when the examples package was published (Because Reasons™️) but is no longer relevant now that we're thankfully not publishing it.